### ISSUE TITLE
`rill init` produces `transport: Error while dialing dial tcp :9081`

### ISSUE THREAD

**Issue by egor-ryashin** [rilldata/rill-developer#1172]
**Describe the bug**
`rill init` produces `transport: Error while dialing dial tcp :9081`, the exact output:
```
Error: connection error: desc = "transport: Error while dialing dial tcp :9081: connect: connection refused"
    at DuckDBClient.request (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:138:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async DuckDBClient.connectRuntime (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:109:21)
    at async DuckDBClient.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:33:9)
    at async DatabaseService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DatabaseService.js:26:9)
    at async DataModelerService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/data-modeler-service/DataModelerService.js:52:9)
    at async RillDeveloper.init (/snapshot/rill-developer/web-local/dist/web-local/src/server/RillDeveloper.js:42:9)
    at async InitCommand.initServerInstances (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:62:9)
    at async InitCommand.init (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:43:13)
    at async InitCommand.run (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:72:9)
```

**To Reproduce**
1. Download https://cdn.rilldata.com/rill/latest/macos-x64/rill
2. Run `rill init`

**Expected behavior**
The project should be initialized.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version 0.13.0

**Workaround**
1. Remove the generated folders. 
2. Run `rill init` once again.

**Comment by magorlick** [comment]
I can't replicate this issue when I use the curl script to bring down Rill. For some reason I can no longer use the manual binary:
```

(base) marissagorlick@MacBook-Pro-2 test4 % /Users/marissagorlick/Downloads/test4/rill-latest init
zsh: permission denied: /Users/marissagorlick/Downloads/test4/rill-latest

```

 I am also on MacOS version 0.13 of Rill Developer. 

@egor-ryashin , do you see this when you use the curl script and use Rill globally?

**Comment by egor-ryashin** [comment]
I used `manual` approach.
@magorlick I used `chmod +x rill` to fix `permission denied`.

**Comment by magorlick** [comment]
@egor-ryashin , thanks for the permissions and quarantine reminder for manual binaries. :) I did those steps and I can't reproduce what you are encountering.

**Comment by egor-ryashin** [comment]
@hamilton I run in an empty directory

**Comment by alexdatadesign** [comment]
Getting the same error:
(ubuntu 22.04)
```
./rill init-example
Initializing the example project /home/axel/rilldata/rill-developer-example ...
Directory /home/axel/rilldata/rill-developer-example already exist. Attempting to initialize the project. 
Error: connection error: desc = "transport: Error while dialing dial tcp :9081: connect: connection refused"
    at DuckDBClient.request (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:138:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async DuckDBClient.connectRuntime (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:109:21)
    at async DuckDBClient.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:33:9)
    at async DatabaseService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DatabaseService.js:26:9)
    at async DataModelerService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/data-modeler-service/DataModelerService.js:52:9)
    at async RillDeveloper.init (/snapshot/rill-developer/web-local/dist/web-local/src/server/RillDeveloper.js:45:9)
    at async InitCommand.initServerInstances (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:62:9)
    at async InitCommand.init (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:43:13)
    at async InitCommand.run (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:72:9)
```

**Comment by magorlick** [comment]
Hi @alexdatadesign, thank you for reporting.  Have you installed Rill multiple times for updates? Could you let us know what version you are on by running `./rill --version` (manual download path) and/or `rill --version` (curl script path).

**Comment by AdityaHegde** [comment]
Looks like the example is an old project, based on the `already exist` message. Sadly older databases cannot always be read by newer duckdb. Can you try running it on a fresh folder?

**Comment by alexdatadesign** [comment]
```
./rill --version
0.14.1
```
Manually removed rill-developer-example

run again 
```
/rill init-example
Initializing the example project /home/axel/rilldata/rill-developer-example ...
Directory /home/axel/rilldata/rill-developer-example doesn't exist. Creating the directory.
Error: connection error: desc = "transport: Error while dialing dial tcp :9081: connect: connection refused"
    at DuckDBClient.request (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:138:19)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async DuckDBClient.connectRuntime (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:109:21)
    at async DuckDBClient.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DuckDBClient.js:33:9)
    at async DatabaseService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/database-service/DatabaseService.js:26:9)
    at async DataModelerService.init (/snapshot/rill-developer/web-local/dist/web-local/src/common/data-modeler-service/DataModelerService.js:52:9)
    at async RillDeveloper.init (/snapshot/rill-developer/web-local/dist/web-local/src/server/RillDeveloper.js:45:9)
    at async InitCommand.initServerInstances (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:62:9)
    at async InitCommand.init (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:43:13)
    at async InitCommand.run (/snapshot/rill-developer/web-local/dist/web-local/src/cli/DataModelerCliCommand.js:72:9)
^C

```

**Comment by AdityaHegde** [comment]
@alexdatadesign Could you gimme more information on what kind of system and OS you have?

**Comment by alexdatadesign** [comment]
@AdityaHegde Ubuntu 22.04 server edition
```
Linux ubuntu22-dev 5.15.0-52-generic #58-Ubuntu SMP Thu Oct 13 08:03:55 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by AdityaHegde** [comment]
Thanks for the info. We will take a look and get back to you.

**Comment by magorlick** [comment]
Hi @alexdatadesign .  We released Rill 0.16 yesterday with some big changes in the way projects work. I just validated that we work in linux on codespaces. Give it another try when you have a chance and let me know if it works for you.

https://docs.rilldata.com/using-rill/install
